### DISCUSSION TITLE
Welcome to Willow Discussions!

### DISCUSSION THREAD

**Discussion by kristiankielhofner** [toverainc/willow#49] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by stiltjack** [comment]
Hi - UK Home Assistant user, ESP Box arrived this morning.

Played with it for an hour or so, and very impressed. One glaring issue, though: how do I connect with it? According to the user guide the initial connection to the Android app is through Bluetooth.  All I get is "Device not found" and there is nothing on a Bluetooth scan. Do I need to do something to make it discoverable, or perhaps there is a timeout ? I notice that there is a cogwheel icon in the top right-hand corner of the screen, but it is unresponsive.

I can connect to the Box's own wi-fi, but there are no controls there that I can see.

Faulty unit?

**Reply by kristiankielhofner** [reply]
Out of the box the ESP BOX comes with a demo from Espressif. You need to (for now) follow the build instructions in the README to configure Willow for your environment (WiFi, HA, etc). Once you flash start talking and you're good to go!

Willow is very early and we understand the process is very technical. We're here to help our early adopters such as yourself to get through it and be one of the first people to enjoy Echo voice assistant quality - without the creepy big tech!

**Comment by stiltjack** [comment]
I undertstand that it's a demo - but I ought to be able to connect the app, wouldn't you say?

**Reply by kristiankielhofner** [reply]
Ignore everything on the ESP BOX screen in front of you or the pamphlet inside the box. We will be erasing all of that - just unplug it for now.

You configure and build the Willow environment (from the README) on your computer to point the ESP BOX you have to your WiFi and Home Assistant. You then plug it in your computer, put Willow on the ESP BOX (flash), and go.
